examples/step-98/step-98.cc

```diff
@@ -0,0 +1,1106 @@
+    {}
+
+    /**
+     * @brief Initialize the patch inverse operator.
+     *
+     * This function computes the 1D mass and stiffness matrices on the
+     * reference cell, scales them according to the mapping information
+     * obtained from the MatrixFree object, assembles the patch matrices
+     * (patch_mass, patch_laplace) by combining contributions from adjacent
+     * cells, and initializes the TensorProductMatrixSymmetricSum object used
+     * for applying the inverse.
+     *
+     * @param mf A shared pointer to the MatrixFree object containing geometric
+     *           and finite element information. Assumes a uniform grid patch.
+     */
+    void initialize(std::shared_ptr<const MatrixFree<dim, number>> mf);
+
+    /**
+     * @brief Apply the inverse of the patch matrix to a vector.
+     *
+     * Solves the linear system `patch_matrix * dst = src` for `dst`.
+     *
+     * @tparam VectorType The type of the source and destination vectors
+     *                    (typically AlignedVector<VectorizedValues>).
+     * @param dst The destination vector (solution).
+     * @param src The source vector (right-hand side).
+     */
+    template <typename VectorType>
+    void vmult(VectorType &dst, const VectorType &src) const;
+
+    /**
+     * @brief Apply the transpose of the inverse patch matrix (equivalent to
+     * vmult for symmetric matrices).
+     *
+     * @param dst The destination vector.
+     * @param src The source vector.
+     * @param tmp_memory Temporary storage (unused in this implementation).
+     */
+    void Tvmult(LinearAlgebra::distributed::Vector<number>       &dst,
+                const LinearAlgebra::distributed::Vector<number> &src,
+                AlignedVector<VectorizedValues> &tmp_memory) const
+    {
+      vmult(dst, src, tmp_memory);
+    }
+
+  private:
+    /**
+     * @brief Shared pointer to the MatrixFree data associated with this patch
+     * operator. Used to extract mapping information.
+     */
+    std::shared_ptr<const MatrixFree<dim, number>> data;
+
+    /**
+     * @brief The tensor product matrix representing the patch operator.
+     * Provides the `apply_inverse` method.
+     */
+    TensorProductMatrixSymmetricSum<dim, VectorizedValues, n_rows_1d>
+      patch_matrix;
+
+    /**
+     * @brief Array storing the 1D mass matrices for each dimension, assembled
+     * over the two reference cells forming the patch dimension.
+     */
+    std::array<Table<2, VectorizedValues>, dim> patch_mass;
+    /**
+     * @brief Array storing the 1D Laplace matrices for each dimension,
+     * assembled over the two reference cells forming the patch dimension and
+     * scaled by mapping factors.
+     */
+    std::array<Table<2, VectorizedValues>, dim> patch_laplace;
+  };
+
+  /**
+   * @brief Initialize implementation.
+   *
+   * Computes 1D reference mass and stiffness matrices, scales them using
+   * Jacobian information from the first cell batch in the MatrixFree object
```

Comment:
I am not sure if a reader will understand what `Jacobian information` refers to. Maybe this is more generic:
```suggestion
   * the metric terms from the first cell batch in the MatrixFree object
```
But feel free to describe it more explicitly as well, e.g. by stating that you get the Jacobian matrix of the transformation to derive the scaling of derivatives by the chain rule.